pkgs/by-name/pa/pace/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  installShellFiles,
+  ...
+}: let
+  version = "0.15.2";
+in
+  rustPlatform.buildRustPackage {
+    pname = "pace";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "pace-rs";
+      repo = "pace";
+      rev = "pace-rs-v${version}";
```

```suggestion
      rev = "refs/tags/pace-rs-v${version}";
```
prevents confusion for update bots esp if an identically-named branch appears